src/UI/Icon.elm

```diff
@@ -291,16 +304,23 @@ defaultOptions =
     }
 
 
+svgIcon : String -> Element msg
+svgIcon iconId =
+    Element.html <|
+        Svg.svg
+            [ SvgAttrs.width "100%"
+            , SvgAttrs.height "100%"
```

Comment:
Can we adopt that all icons will be 1:1, and replace `height:100%` with preversing aspect-ratio?